.github/workflows/ci.yml

```diff
@@ -9,18 +9,20 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Setup Scala
-        uses: olafurpg/setup-scala@v10
+        uses: olafurpg/setup-scala@v14
         with:
-          java-version: "adopt@1.11"
+          java-version: "openjdk@1.17"
       - name: Coursier cache
         uses: coursier/cache-action@v5
       - name: Build and test
         run: |
           sbt -v -J-Xms2048M -J-Xmx2048M -J-Xss6M -J-XX:ReservedCodeCacheSize=256M -J-Dfile.encoding=UTF-8 clean scalafmtCheckAll test scripted
+        env:
+          PROTOFETCH_GIT_PROTOCOL: https
```

Comment:
Why https ?

Reply:
It will not work as smoothly as ssh IMO
https://github.com/coralogix/protofetch#https-support

Reply:
that's what protodep was using until now, also without this protofetch is not able to fetch and we'd need to set up ssh just for scripted tests

Reply:
so we can either keep using protodep, which nobody is using anyway, or do this for now and use protofetch in those scripted tests

Reply:
This is only for CI yes ?

Reply:
yes, just to make this CI step to pass :)

Reply:
Lmao. Ignore what I said then. sorry